packages/text/src/styledComponents/Caption.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+
+const CaptionContainer = styled.div`
+  font-family: Lato, sans-serif;
```

Comment:
Maybe we should include line-height here (and on the others), Zeplin shows different line heights for the different sizes